app-ng/src/job.lisp

```diff
@@ -1,5 +1,13 @@
+;;;; app-ng/src/job.lisp
+;;;;
+;;;; author: Mark Skilbeck
+;;;;         appleby
```

Comment:
Not a fan of using your full name?

Reply:
I'm like a Brazilian footballer.  I can change yours to Marquinho if u want.